### PR TITLE
Pair Programming: Update JavaScript code to use classes 

### PR DESCRIPTION
**Pull request by Lasten-Ernest**
This pull request for setting up the Awesome book's website using JavaScript classes. This means that we will refactor the code to use the JavaScript classes. The screenshoot how the project look like is below:
Sd

### CODE REVIEW

**Review by Davidosky007 — COMMENTED** [review]
Hi @Lasten-Ernest @Piercel2022 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesnt show equal ammount of contribution from your partner, as this is a paired programming project you are expected to work together on this project. you should both commit codes to this project 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script/addRemoveBooks.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    return books;
+  }
+
+  static storeBook(newBook) {
+    const books = Book.getBooks();
+    books.unshift(newBook);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(bookId) {
```
Comment (muneebulrehman) [inline script/addRemoveBooks.js]: Your remove functionality is not working properly. It is not deleting the exact element I want it to. Kindly fix it.
If you need any further explanation, let me know on slack.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Lasten-Ernest , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 

Hi @team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters :heavy_check_mark: 
- [ ] Good commit history :heavy_check_mark: 
- [ ] Descriptive pull request :heavy_check_mark: 
- [ ] Good implementation of changes required :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**